src/include/cdb/cdbappendonlyblockdirectory.h

```diff
@@ -322,4 +346,35 @@ copy_out_minipage(MinipagePerColumnGroup *minipageInfo,
 	minipageInfo->numMinipageEntries = minipageInfo->minipage->nEntry;
 }
 
+static inline void
+AOBlkDirScan_Init(AOBlkDirScan blkdirscan,
+				  AppendOnlyBlockDirectory *blkdir)
+{
+	blkdirscan->blkdir = blkdir;
+	blkdirscan->sysscan = NULL;
+	blkdirscan->segno = -1;
+	blkdirscan->colgroupno = 0;
+}
+
+/* should be called before fetch_finish() */
+static inline void
+AOBlkDirScan_Finish(AOBlkDirScan blkdirscan)
+{
```

Comment:
Are these functions used multiple times? If not, the details should be hidden.

Reply:
AOBlkDirScan_Init used by `aocaam.c` and `appendonlyam.c`.